virttest/test_setup.py

```diff
@@ -1286,7 +1288,7 @@ def get_same_group_devs(self, pci_id):
 
     def get_pf_mlx(self):
         """
-        Get the PF devices for mlx5_core driver
+        Get the ifname of PF devices
```

Comment:
In this PR, this function was not used for mlx5_core driver only, so how about change the function name too ?  and in fact, i prefer to delete this function,  its function is repeated with get_pf_ids partly.

Reply:
Good idea to update the function name. Actually, this function is useful when setting ip for the interface(L1500).

Reply:
Recommended to get interface name with "ls /sys/bus/pci/devices/0000\:04\:00.0/net/". refer the comment  for set_vf()